astropy/coordinates/representation.py

```diff
@@ -496,6 +701,33 @@ def from_representation(cls, representation):
         """
         return representation.represent_as(cls)
 
+    def _apply(self, method, *args, **kwargs):
+        """Create a new representation with ``method`` applied to the component
+        data.
+
+        This is not a simple inherit from ``BaseRepresentationOrDifferential``
+        because we need to call ``._apply()`` on any associated differential
+        classes.
+
+        See docstring for `BaseRepresentationOrDifferential._apply`.
+
+        Parameters
+        ----------
+        method : str or callable
+            If str, it is the name of a method that is applied to the internal
+            ``components``. If callable, the function is applied.
+        args : tuple
+            Any positional arguments for ``method``.
+        kwargs : dict
+            Any keyword arguments for ``method``.
+
+        """
+        rep = super(BaseRepresentation, self)._apply(method, *args, **kwargs)
+        diffs = [diff._apply(method, *args, **kwargs)
```

Comment:
Just go for a tuple directly by "tuple comprehension": `tuple(diff._apply(...) for ...)`